src/v2/Apps/Consign/ConsignApp.tsx

```diff
@@ -9,7 +9,7 @@ import { SellArtDifferently } from "./Components/SellArtDifferently"
 import { GetPriceEstimate } from "./Components/GetPriceEstimate"
 import { HowToSell } from "./Components/HowToSell"
 import { ConsignInDemandNow } from "./Components/InDemandNow/ConsignInDemandNow"
-import { SoldRecently } from "./Components/SoldRecently"
+import { SoldRecentlyQueryRenderer as SoldRecently } from "./Components/SoldRecently"
```

Comment:
Note that in Eigen we've stopped aliasing QueryRenderer's / fragmentContainers, and there's an RFC open for doing the same in force, which is likely to pass: https://github.com/artsy/README/issues/335